t/api/15-users.t

```diff
@@ -31,4 +31,7 @@ $t->ua(OpenQA::Client->new(apikey => 'LANCELOTKEY01', apisecret => 'MANYPEOPLEKN
 $t->app($app);
 $t->delete_ok('/api/v1/user/99904')->status_is(403, 'non-admins cannot delete users');
 
+$t->post_ok('/api/v1/feature?version=42')->status_is(200, 'can set the feature version of current user');
+is $app->schema->resultset('Users')->find(99902)->feature_version, 42, 'feature version was updated';
```

Comment:
kinda nitpicking but I think the test checks the value in the DB. it doesnt update  it. I mean I understand that it reffers to the previous action but maybe rephrase it. 
Something like `Schema returns correct feature version` or `Correct feature version value can be found in database`. wdyt?

Reply:
It is a post, though? And it checks the new version

Reply:
I would not mention the database explicitly here. If we did this in similar cases we would lengthen a lot of lines in our code. Note that the test names are also generally supposed to explain the check on a higher level instead of just repeating what the code does.

I think using the past here makes it clear enough that the updating was done before the check and the check is just verifying what happened before (in this case the post).